config/default.yaml

```diff
@@ -68,8 +68,8 @@ noop: false
 
 # Set to true to use /joined_members instead of /state to figure out who is
 # in the room. Using /state is preferred because it means that users are
-# banned when they are invited instead of just when they join, though if your
-# server struggles with /state requests then set this to true.
+# banned when they are invited instead of just when they join. Set this to true
+# if the bot is in large rooms or dozens of rooms.
```

Comment:
It's a nitpick but shouldn't we say both of these things? Something like "If your server struggles with /state requests or the bot is in large or dozens of rooms set this to true"?

Really it's up to you though

Reply:
I'm not sure how a user would know if their server struggles with /state requests. Is there a way we could tell them how to know?

Reply:
Good question, I think they can through a combination [grafana](https://matrix-org.github.io/synapse/latest/usage/administration/understanding_synapse_through_grafana_graphs.html) or [prometheus](https://matrix-org.github.io/synapse/latest/metrics-howto.html) and maybe also with synapse [workers](https://matrix-org.github.io/synapse/latest/workers.html) but I am not entirely sure.

Reply:
Maybe "...when they join, though if your server struggles with /state requests (which is likely the case if the bot is in large rooms or dozens of rooms), then set this to true."